docs/source/model_design/risk_factors.rst

```diff
@@ -531,9 +531,206 @@ Difference measures relating to the whole population tend to be more useful and
+eliminated by removing a risk exposure.
+
+..todo::
+	Fill in PAF overview
+
+Bias in PAF Calculation
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The PAF can be calculated using the following formula:
+
+.. math::
+	:label: eq1
+
+	\text{PAF}_{pop}=\frac{p_c(RR_{adj}-1)}{RR_{adj}}
+
+In which we define :math:`p_c` to be the proportion of cases (individuals who 
+possess the outcome of interest) that are exposed, and :math:`RR_{adj}` has been adjusted for confounding and effect modification.
+
+There is the a second PAF equation, which can be used *in the absence of 
+confounding or effect modification:*
+
+.. math:: \text{PAF}_{pop}=\frac{p_p(RR_{cr}-1)}{1+p_p(RR_{cr}-1)} =\frac{p_p(RR_{adj}-1)}{1+p_p(RR_{adj}-1)}
+	:label: eq2
+
+Note that here, the crude relative risk :math:`(RR_{cr})` is equivalent to the adjusted :math:`(RR_{adj})`. We define :math:`p_p` to be the proportion of the entire 
+population that is exposed.
+
+This is typically easier to conceptualize if we break the population down as
+follows:
+
+.. list-table:: Exposure x Cases
+	:widths: 1 1 1
+	:header-rows: 1
+	:stub-columns: 1
+	:align: center
+
+	* - 
+	  - Cases
+	  - Non-cases
+	* - Exposed
+	  - a
+	  - b
+	* - Unexposed
+	  - c
+	  - d
+
+Observe that the above table is a full partition of our population. We can see 
+then that the proportion of cases that are exposed is given by:
+
+..	math:: p_c=\frac{a}{a+c}
+
+And the proportion of the entire population that is exposed is given by:
+
+.. math:: p_p = \frac{a+b}{a+b+c+d}
+
+It can be shown that when the fraction of cases in the unexposed times the 
+relative risk :math:`\left( \frac{c}{c+d} \cdot RR_{adj} \right)` equals the fraction 
+of cases in the exposed :math:`\left( \frac{a}{a+b} \right)`, i.e., when there 
+are no confounders, equation (1) equals equation (2).
+
+However, when :math:`\frac{c}{c+d} \cdot RR_{adj} \neq \frac{a}{a+b}`, this inequality 
+fails. Intuitively, we can imagine a confounder that is positively associated with
+our exposure, holding all else constant. Then there will be a 
+disproportionately high number of cases among the exposed, and 
+:math:`\frac{c}{c+d} \cdot RR_{adj} < \frac{a}{a+b}`.
+
+This can be solved via weighting equation (2) per stratum of our confounder or
+effect modifier, yielding equation (3):
+
+.. math:: \sum_{i=1}^z W_i \frac{p_i(RR_i-1)}{1+p_i(RR_i-1)}
+	:label: eq3
+
+Here, for each stratum :math:`i` of our confounder or effect modifier, 
+:math:`p_i` is the proportion of the stratum that is exposed, :math:`W_i` is 
+the proportion of the cases in the stratum, and :math:`RR_i` is the 
+stratum-specific adjusted *RR*. Note that in the case of a confounder, 
+:math:`RR_{i}` will be equal across strata, and in the case of effect 
+modification, there will be a different :math:`RR_{i}` per stratum. More 
+information on confounding and effect modification can be found
+in the section on :ref:`causal relationships<causal_relationships>`.
```

Comment:
Could someone look at this and confirm that it's true/correct?

Reply:
Which part are you wondering about, specifically?  The equation itself or the statement about equal vs. unequal RR_i's for confounding vs. effect modification?  I'll have to double-check the references and think through it to be sure.

Reply:
@NathanielBlairStahn  I am confused about why the Darrow paper has RR_i instead of just RR.

I am thinking that in the case that there is NO effect modification, RR_i = RR_j for all i,j\in stratum.

And thus RR_i only needs to be introduced for effect modification.